service/tls.go

```diff
@@ -69,6 +70,12 @@ func NewTLSConfig(cfg config.TLS) (*tls.Config, error) {
 // Server and CA Certificate, and private key will read from files from file paths defined in environment variables.
 func NewTLSConfigWithTLSCertificateCache(cfg config.TLS) (*tls.Config, *TLSCertificateCache, error) {
 	var tcc *TLSCertificateCache
+
+	cs, err := cipherSuites(cfg.DisableCipherSuites)
+	if err != nil {
+		return nil, nil, errors.Wrap(err, "cannot cipherSuite(cfg.DisableCipherSuites)")
```

Comment:
```suggestion
		return nil, nil, errors.Wrap(err, "cipherSuite(cfg.DisableCipherSuites)")
```
Just found a weird error message. Can you fix below error too?

https://github.com/AthenZ/authorization-proxy/blob/9365b69727228bd3b1cedcf9ccd09dd17ecf7611/service/tls.go#L98

Reply:
@WindzCUHK Fixed